src/main/java/com/okta/sdk/framework/ApiClient.java

```diff
@@ -39,6 +40,7 @@
     protected String token;
     protected ApiClientConfiguration configuration;
     private final Logger LOGGER = LoggerFactory.getLogger(ApiClient.class);
+    private HttpResponse lastHttpResponse;
```

Comment:
My only question is whether this should be `static`, so the same RateLimitContext is available across all ApiClients